Casks/v/virtualdj.rb

```diff
@@ -0,0 +1,31 @@
+cask "virtualdj" do
+  version "2024.b8124"
```

Comment:
```suggestion
  version "2024,8124"
```

Split the version number with a comma between the version and then the build number.